### PR TITLE
image search kit batch inference job and online inference ingestion modes fixed

### PR DESCRIPTION
**Pull request by snova-jorgep**

### CODE REVIEW

File: image_search/notebooks/image_search.ipynb
Comment (snova-codym) [inline image_search/notebooks/image_search.ipynb:1]: In the notebook, I see:

# create dataset
        command = f'echo yes | snapi dataset add \
            --dataset-name {dataset_name} \
            --job_type {self.job_type} \
            --apps {self.clip_app_id} \
            --source_type {self.dataset_source_type} \
            --source_file {self.dataset_source_file} \
            --description "{self.dataset_description}"'
            
Are the nested double quotes required?

Comment (snova-codym) [reply image_search/notebooks/image_search.ipynb:1]: Typo?

logging.info(f'Project {self.project_name} wasn\'t found in SambaStudio')

Comment (snova-jorgep) [reply image_search/notebooks/image_search.ipynb:1]: yes those nested quotes are required for description, also typos fixed

File: image_search/src/image_search.py [hunk 1]
```diff
@@ -103,29 +102,33 @@ def get_images(self, folder_path = None ):
         print(f"got {len (images)} images")
         return paths,images
 
-    def add_images(self, path, batch=False):
+    def add_images(self, path = None):
+        if path is None:
+            path=os.path.join(kit_dir,"data/images")
```
[the remark this answers is not in the document]
Comment (snova-jorgep) [reply image_search/src/image_search.py]: right adjusted here and also in line 92

**Review by snova-codym — APPROVED** [review]

**Pull request merged**